docs/docsite/rst/roadmap/COLLECTIONS_12.rst

```diff
@@ -25,18 +25,18 @@ The schedule has been updated to match the :ref:`ansible-core 2.19 release sched
 :2025-04-14: ansible-core feature freeze, stable-2.19 branch created.
 :2025-04-14: Start of ansible-core 2.19 betas.
 :2025-04-16: Ansible-12.0.0 alpha1; there will be more alpha releases usually following additional ansible-core 2.19 betas.
-:2025-05-26: First ansible-core 2.19 release candidate.
-:2025-05-27: Ansible-12.0.0 alphaX; there might be more alpha releases following additional ansible-core 2.19 release candidates.
-:2025-06-16: Ansible-core-2.19.0 released.
-:2025-06-16: Last day for collections to make backwards incompatible releases that will be accepted into Ansible-12. This includes adding new collections to Ansible 12.0.0; from now on new collections have to wait for 12.1.0 or later.
-:2025-06-17: Ansible-12.0.0 beta1 -- feature freeze [2]_ (collection owners and interested users should test for bugs).
-:2025-06-24: Ansible-12.0.0 rc1 [3]_ [4]_ (weekly release candidates as needed; test and alert us to any blocker bugs). Blocker bugs will slip release.
-:2025-06-27: Last day to trigger an Ansible-12.0.0rc2 release because of major defects in Ansible-12.0.0rc1.
-:2025-07-01: Ansible-12.0.0rc2 when necessary, otherwise Ansible-12.0.0 release.
-:2025-07-08: Ansible-12.0.0 release when Ansible-12.0.0rc2 was necessary.
-:2025-07-01 or 2023-07-08: Create the ansible-build-data directory and files for Ansible-13.
-:2025-07-14: Release of ansible-core 2.19.1.
-:2025-07-15: Release of Ansible-12.1.0 (bugfix + compatible features: every four weeks.)
+:2025-07-??: First ansible-core 2.19 release candidate.
+:2025-07-??: Ansible-12.0.0 alphaX; there might be more alpha releases following additional ansible-core 2.19 release candidates.
```

Comment:
```suggestion
:2025-06-30: First ansible-core 2.19 release candidate.
:2025-07-01: Ansible-12.0.0 alphaX; there might be more alpha releases following additional ansible-core 2.19 release candidates.
```

#2679 hasn't been merged yet, but if it is I think this would be the wayy to go.

Reply:
Ah, good to know that there's a PR now.